astrology.lic

```diff
@@ -329,8 +329,10 @@ class Astrology
         check_weather
       when 'events'
         check_events(check_pools)
-      when 'moons'
-        check_moons
+      when 'moons'  #deprecated - left for backwards compat. only
```

Comment:
Oh, good catch. I think we should add a validator for `astrology_training`. Or maybe two - one that checks for valid values and one for the deprecated 'moon' value. What do you think?

Reply:
ill work on that this weekend. Also, the above modifications are incorrect and do not correctly check pool size. It needs a parenthesis set. Please do not merge.